utilities/radius_separated_points.m

```diff
@@ -0,0 +1,23 @@
+function ind = radius_separated_points(long,lat,radius)
+%  ind = radius_separated_points(long,lat,radius)
+%  get the indices of points in [long, lat] to keep 
+%  so that they are at least separated by a distance of
+%  user-defined 'radius'
+
+% computing rangesearch
```

Comment:
Please use ourRangeSearch to avoid toolbox dependency if possible

Reply:
ourRangeSearch doesn't seem to work for multiple points.

Reply:
you call it in a loop over all the query points.

Reply:
see this line there's some unfortunately hardcoded parameters. 
[idx,~] = frsearch(anno, testset, r, 1e3,0,1);

Reply:
ok updating that now..

Reply:
latest commit addresses this

Reply:
but note for test I was using it took 3 sec for rangesearch but 50 sec for ourRangeSearch.

Reply:
yea, it's not vectorized. I guess you use this for quiver plotting. What you could do is check if the rangesearch exists if they have the toolbox and then default to this.

Reply:
yeah it does do that